entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md

```diff
@@ -0,0 +1,125 @@
+---
+title: What's New in EF Core 5.0
+author: ajcvickers
+ms.date: 01/29/2020
+uid: core/what-is-new/ef-core-5.0/whatsnew.md
+---
+
+# What's New in EF Core 5.0
+
+EF Core 5.0 is currently in development.
+This page will contain an overview of interesting changes introduced in each preview.
+The first preview of EF Core 5.0 is tentatively expected in the next couple of months.
+
+This page does not duplicate the [plan for EF Core 5.0](plan.md).
+The plan describes the overall themes for EF Core 5.0, including everything we are planning to include before shipping the final release.
+
+We will add links from here to the official documentation as it is published.
+
+## Preview 1 (Not yet shipped)
+
+### Simple logging
+
+This feature is the moral equivalent of `Database.Log` in EF6.
```

Comment:
I didn't realize that `Database.Log` had morality. Consider "offers similar functionality"

Reply:
Fine.